### ISSUE TITLE
Save content saves only visible screen

### ISSUE THREAD

**Issue by filipjenicek** [Guake/guake#1958]
**Describe the bug**
The "Save content..." option in the context menu saves only the visible content of the terminal.

**Expected behavior**
Full terminal output is saved including scrolled content.

**Actual behavior**
On Debian 11, only the visible part is saved.
AFAIK on Debian 10 all contents were saved correctly.

**To Reproduce**
Debian 11 (Oct29) xfce4
Reproduced with debian packages 3.4.0-1, 3.8.0-1 and also with the latest src code a084a1d0d11d2306ca5447c2cfaf31db0393c798.

<details><summary>$ guake --support</summary>

Guake Version:		3.8.2.dev4

Vte Version:		0.62.3

Vte Runtime Version:	0.62.3

--------------------------------------------------
GTK+ Version:		3.24.24

GDK Backend:		<GdkX11.X11Display

--------------------------------------------------
Desktop Session: lightdm-xsession

--------------------------------------------------
Display: :0.0

RGBA visual: True

Composited: True

* Monitor: 0 - DEL DVI-0
    * Geometry:		1920 x 1200 at 0, 0
    * Size:		518 x 324 mm²
    * Primary:		True
    * Refresh rate:	59.95 Hz
    * Subpixel layout:	horizontal-rgb

**Comment by filipjenicek** [comment]
You can reproduce it on Debian 11:
1. Download https://www.linuxvmimages.com/images/debian-11/
2. #sudo apt install guake
3. open guake, execute sudo dmesg
4. Run the guake context menu action "save content..."
5. The resulting file has only a few lines depending on your screen size.

It works well on Debian 10:
https://www.linuxvmimages.com/images/debian-10

The key difference is the version of libvte, which is 0.62.3 on Debian 11, and 0.54.2-2 on Debian 10.

If you downgrade libvte to 0.54.2-2 (https://packages.debian.org/buster/libvte-2.91-0), it starts to work well. I suppose the guake's clipboard hack is not compatible with newer releases of libvte.

**Comment by Davidy22** [comment]
Don't have a newer version of VTE in my repo, but since this is isolated to newer VTE I should be able to figure this out, can start checking documentation for now.

**Comment by Davidy22** [comment]
Now have vte 0.66.2 and still not encountering this, although I am observing some kind of hang that's probably a different issue.

**Comment by nyaapa** [comment]
Reproes on ubuntu 22.04 w/ both stock (3.8.5) and updated from linuxuprising (3.10) guake.
```
➤  dpkg -l | grep vte  
ii  gir1.2-vte-2.91:amd64                      0.68.0-1                                   amd64        GObject introspection data for the VTE library
ii  libvte-2.91-0:amd64                        0.68.0-1                                   amd64        Terminal emulator widget for GTK+ 3.0 - runtime files
ii  libvte-2.91-common                         0.68.0-1                                   amd64        Terminal emulator widget for GTK+ 3.0 - common files
```

**Comment by smith-doug** [comment]
Based on the above comment https://github.com/smith-doug/guake/commit/156a30ba8d2e4fcbc86bc8da186f5c59cbbc174e
This seems to work, doesn't miss any text, and respects clear.  I'm on Ubuntu 22.04.  

At some point Vte changed the behavior of select_all.  This has affected other terminals https://github.com/mate-desktop/mate-terminal/issues/372

**Comment by PhungXuanAnh** [comment]
I also faced this error in ubuntu 22.04, I applied the change in this PR https://github.com/Guake/guake/pull/2218 to fix this issue

**Comment by Davidy22** [comment]
Closing, resolved in #2218
